boilerplates/lucia-auth/files/server/lucia-auth-handlers.ts

```diff
@@ -0,0 +1,303 @@
+import { generateId, verifyRequestOrigin, type Session } from "lucia";
+import { github, lucia } from "../libs/auth";
+import { hash, verify, type Options } from "@node-rs/argon2";
```

Comment:
Native module, I'm not sure it's compatible with all serverless solutions

Reply:
Quoted from https://lucia-auth.com/guides/email-and-password/basics#hashing-passwords

Argon2id should be your first choice for hashing passwords, followed by Scrypt and Bcrypt. Hashing is by definition computationally expensive so you should use the most performant option for your runtime.
- For Node.js we recommend using [@node-rs/argon2](https://github.com/napi-rs/node-rs).
- For Bun, we recommend using [Bun.password](https://bun.sh/docs/api/hashing).
- Use Deno-specific packages for Deno.
- For other runtimes (e.g. Cloudflare Workers), your choice is very limited. [@noble/hashes](https://github.com/paulmillr/noble-hashes) provides pure-js implementations of various hashing algorithms, but because it's written in JS, you may hit into CPU limitations of your service. If possible, avoid these runtimes when you need to hash passwords.

Reply:
A pure JS implementation of Scrypt built in `lucia` package : [Scrypt](https://lucia-auth.com/reference/main/Scrypt/)

Reply:
> A pure JS implementation of Scrypt built in `lucia` package : [Scrypt](https://lucia-auth.com/reference/main/Scrypt/)

Seems good enough for now. A comment redirecting to the documentation regarding perfs should be enough